ui/src/dashboards/containers/DashboardPage.tsx

```diff
@@ -532,6 +523,30 @@ class DashboardPage extends Component<Props, State> {
 
     this.setState({scrollTop: target.scrollTop})
   }
+
+  private get dashboardLinks(): DashboardsModels.DashboardSwitcherLink[] {
+    const {dashboards, source} = this.props
+
+    return dashboards.map(d => {
+      return {
+        id: String(d.id),
```

Comment:
hrrm... having just typed dashboard ids from numbers to strings recently, (the coercion horror!) I got a little sad seeing that the id type was a string. Could it remain a number for comparison to dashboard id's but be changed in to a string if it needs to be used as a string down the line... not sure about this, just a lingering sadness.

Reply:
yeah I see what you mean, but I do like the consistency of all identifiers throughout the app being strings. In this case it's not used as the `id` of a dashboard, but rather as something to distinguish two `DashboardSwitcherLink`s in a list. I renamed the `id` to `key` to make this more clear and to hopefully avoid confusion as to what this property signifies.